src/goto-programs/validate_goto_model.cpp

```diff
@@ -0,0 +1,251 @@
+/*******************************************************************\
+
+Module: Validate Goto Programs
+
+Author: Diffblue
+
+Date: Oct 2018
+
+\*******************************************************************/
+#include "validate_goto_model.h"
+
+#include <set>
+
+#include "goto_functions.h"
+#include <util/invariant.h>
```

Comment:
Swap the order of these includes and insert a blank line in between.

Reply:
Done.